ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -20,7 +20,7 @@ namespace ReactNative.UIManager
 {
     /// <summary>
     /// Base class that should be suitable for the majority of subclasses of <see cref="IViewManager"/>.
-    /// It provides support for base view properties such as opacity, etc.
+    /// It provides support for base view props such as opacity, etc.
```

Comment:
@rozele What's the deal with props vs properties ?

Reply:
It's confusing to call them properties. The name for the unidirectional data flow in React.js is `prop`, so we should be consistent.